web/src/beta/features/PluginPlayground/Plugins/presets/useTitles.ts

```diff
@@ -59,7 +59,7 @@ export default () => {
       "zoom-in-out": t("Zoom In Out"),
       "camera-rotation": t("Camera Rotation"),
       "camera-position": t("Camera Position"),
-      "shared-plugin-id": t("Shared Plugin")
+      "e99982f9-143a-44db-9869-b2bd90578190": t("Shared Plugin") // NOTE: needed to hardcode this part to help translation
```

Comment:
```suggestion
      "shared-plugin": t("Shared Plugin") // NOTE: needed to hardcode this part to help translation
```
Could we use this instead of a hardcoded uuid?

Reply:
done